### PR TITLE
New Pull Request for the Assignement 4

### PR DESCRIPTION
**Pull request by BaptCarr**
Hello Sir,

You told me to correct the PR for the assignement 4.
I had an issue with the other branch so I only could correct it on this branch. 
Also, as I said before, I don't really know my student number so I put a number I found that seems to be the good one. 

Sincerely,
BC.

### CODE REVIEW

File: Assignment4/course_materials/python/BaptCarr-2023502759016/task07.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+"""**TASK 7.2: List all individuals of "Person" with RDFLib and SPARQL (remember the subClasses)**
+
+"""
+
+# TO DO
+query = prepareQuery("""
+    SELECT ?individual
+    WHERE {
+        ?individual rdf:type ns:Person.
```
Comment (SerPablo) [inline Assignment4/course_materials/python/BaptCarr-2023502759016/task07.py:52]: Remember here the subclasses

File: Assignment4/course_materials/python/BaptCarr-2023502759016/task07.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+from rdflib.plugins.sparql import prepareQuery
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example6.rdf", format="xml")
+
+ns =Namespace("http://somewhere#")
+foaf = Namespace("http://xmlns.com/foaf/0.1/")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0/")
+
```
Comment (SerPablo) [inline Assignment4/course_materials/python/BaptCarr-2023502759016/task07.py:28]: All the exercises have to be in RDFLib and SPARQL

File: Assignment4/course_materials/python/BaptCarr-2023502759016/task07.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        ?individual rdf:type ns:Person.
+    }
+""", initNs={'ns': ns})
+
+# Visualize the results
+
+for r in g.query(query):
+  print(r.individual)
+
+"""**TASK 7.3: List all individuals of "Person" or "Animal" and all their properties including their class with RDFLib and SPARQL. You do not need to list the individuals of the subclasses of person**
```
Comment (SerPablo) [inline Assignment4/course_materials/python/BaptCarr-2023502759016/task07.py:61]: The values of the properties of the individuals

**Review by SerPablo — COMMENTED** [review]
Review comments for task 7

**Comment by anaigmo** [comment]
Submitted and fixed, no more PRs allowed. The submission does not follow the instructions, so it will not be merged

**Pull request closed without merging**